pom.xml

```diff
@@ -25,9 +25,11 @@
     <repo.ajdt.neon>http://download.eclipse.org/tools/ajdt/46/dev/update</repo.ajdt.neon>
     <repo.eclipse.oxygen>http://download.eclipse.org/releases/oxygen</repo.eclipse.oxygen>
     <repo.ajdt.oxygen>http://download.eclipse.org/tools/ajdt/47/dev/update</repo.ajdt.oxygen>
+    <repo.eclipse.photon>http://download.eclipse.org/releases/photon</repo.eclipse.photon>
+    <repo.ajdt.photon>http://download.eclipse.org/tools/ajdt/47/dev/update</repo.ajdt.photon>
```

Comment:
Just noticed this is using the 4.7 site for AJDT, not 4.8. Could this be related to the issue mentioned in https://github.com/scala-ide/scala-ide/pull/1189#issuecomment-413801851?

Reply:
Quite possible. 4.8 update site for AJDT was not yet available at the time I was doing my changes.